### PR TITLE
Add ERC: Ownership Delegation and Context for ERC-721

### PR DESCRIPTION
**Pull request by ducthotran2010**
This proposal introduces an extension for ERC721 Non-Fungible Tokens (NFTs): specifying users for several contexts and differentiating owner and ownership rights for a token to use in several financial use cases for NFT.

### CODE REVIEW

File: ERCS/erc-draft.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-draft.md:1]:
```suggestion
---
eip: 7695
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: ERCS/erc-draft.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+title: Ownership Delegation and Context for Non-Fungible Token
+description: Extends ERC-721 for financial use cases
+author: Duc Tho Tran (@ducthotran2010)
+discussions-to: https://ethereum-magicians.org/t/add-erc-ownership-delegation-and-context-for-non-fungible-token/19716
```
Comment (abcoathup) [inline ERCS/erc-draft.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7695-ownership-delegation-and-context-for-non-fungible-token/19716
```

**Comment by ducthotran2010** [comment]
Hi reviewers, all the commented issues are resolved. This proposal is refined to be a draft and ready for review. Please take a look @axic, @g11tech, @SamWilsn, @xinbenlv

File: ERCS/erc-7695.md [hunk 3]
```diff
@@ -0,0 +1,477 @@
+---
+eip: 7695
+title: Ownership Delegation and Context for ERC-721
+description: Extends ERC-721 for financial use cases without changing owner
```
Comment (SamWilsn) [inline ERCS/erc-7695.md:4]: Your description is a bit vague. Think you could expand it a bit?

Maybe something about creating different contexts, and delegating permissions within those contexts, without changing the ultimate owner of the token?

File: ERCS/erc-7695.md [hunk 4]
```diff
@@ -0,0 +1,477 @@
...
+author: Duc Tho Tran (@ducthotran2010)
+discussions-to: https://ethereum-magicians.org/t/erc-7695-ownership-delegation-and-context-for-non-fungible-token/19716
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-02
+requires: 165, 721
+---
+
+## **Abstract**
```
Comment (SamWilsn) [inline ERCS/erc-7695.md:14]:
```suggestion
## Abstract
```

File: ERCS/erc-7695.md [hunk 5]
```diff
@@ -0,0 +1,477 @@
...
+
+This proposal introduces an extension for [ERC-721](./eip-721.md) Non-Fungible Tokens (NFTs): specifying users for several contexts and differentiating owner and ownership rights for a token to use in several financial use cases for NFT without changing owner.
```
Comment (SamWilsn) [inline ERCS/erc-7695.md:16]: Your abstract is pretty good, except maybe a tad too short. What differentiates your proposal from other similar ones? How does it work on a technical level?

File: assets/erc-7695/mortgage.png
Comment (SamWilsn) [inline assets/erc-7695/mortgage.png:1]: We have a slight preference for SVG images, but if you cannot provide one, this PNG is fine.

File: ERCS/erc-7695.md (hunk 3 above)
Comment (ducthotran2010) [reply ERCS/erc-7695.md]: It was changed to _"Introduces contexts and ownership delegation for ERC-721 tokens, expanding dApps and financial use cases without transferring ownership"_ in this commit https://github.com/ethereum/ERCs/pull/391/commits/064e1e7b60cb8275546c5c2649271a3a7428a9dc
Let me know if I can make it better. Thank you.

File: ERCS/erc-7695.md (hunk 4 above)
Comment (ducthotran2010) [reply ERCS/erc-7695.md]: It is resolved

File: ERCS/erc-7695.md (hunk 5 above)
Comment (ducthotran2010) [reply ERCS/erc-7695.md]: It was updated to:
_"This standard is an extension for ERC-721, designed to specify users for various contexts with a locking feature and allow temporary ownership delegation without changing the original owner._

_This EIP preserves the benefits and rights of the owner while expanding the utility of NFTs across various dApps by adding the concepts of Ownership Delegation and Contexts, which define specific roles: Controller and User, who can use the NFT within defined contexts."_ in this commit https://github.com/ethereum/ERCs/commit/064e1e7b60cb8275546c5c2649271a3a7428a9dc
Let me know if I can make it better. Thank you.

File: assets/erc-7695/mortgage.png
Comment (ducthotran2010) [reply assets/erc-7695/mortgage.png:1]: SVG images are used as your recommendation in this commit https://github.com/ethereum/ERCs/pull/391/commits/bc89fe95341ffc859a61dbc28334a520bfe2e882

**Comment by ducthotran2010** [comment]
Hi reviewers,
I resolved all the commented issues. It is ready to review as a draft proposal, please take a look @axic, @g11tech, @SamWilsn, @xinbenlv, @abcoathup. I will add the reference implementation in the Update ERC PR while determining the best practices for this proposal.
Thank you.

**Pull request merged**